truffle/contracts/Load/LoadContract.sol

```diff
@@ -1,12 +1,12 @@
-pragma solidity 0.5.0;
+pragma solidity 0.5.11;
 
 import {Ownable} from "openzeppelin-solidity/contracts/ownership/Ownable.sol";
 import {ERC20} from "openzeppelin-solidity/contracts/token/ERC20/ERC20.sol";
 
 import {Shipment} from "./lib/Shipment.sol";
-import {Vault} from "./lib/Vault.sol";
 import {Escrow} from "./lib/Escrow.sol";
 import {Converter} from "./lib/Converter.sol";
+//import {VaultNotary} from "../VaultNotary/VaultNotary.sol";
```

Comment:
Let's kill commented code! I used to like to leave commented code around myself, but thanks to `git`, we can always view old code from prior commits if we need to.